man/ronin-unarchive.1.md

```diff
@@ -0,0 +1,33 @@
+# ronin-unarchive 1 "2024-01-18" Ronin "User Manuals"
+
+## NAME
+
+ronin-unarchive - Unarchive the files.
+
+## SYNOPSIS
+
+`ronin unarchive` [*FILE* ...]
+
+## DESCRIPTION
+
+Unarchive the files.
+
+## ARGUMENTS
+
+[*FILE* ...]
```

Comment:
Remove the `[` and `... ]`. Argument definitions are about the individual argument values.